checks/evaluation/pinned_dependencies_test.go

```diff
@@ -228,602 +229,180 @@ func Test_createScoreForGitHubActionsWorkflow(t *testing.T) {
-				},
-			},
-			expected: scut.TestReturn{
-				Error:         nil,
-				Score:         0,
-				NumberOfWarn:  2,
-				NumberOfInfo:  2,
-				NumberOfDebug: 0,
-			},
-		},
-		{
-			name: "GitHub Actions ecosystem with GitHub-owned pinned",
-			dependencies: []checker.Dependency{
-				{
-					Location: &checker.File{
-						Snippet: "actions/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675",
+			findings: []finding.Finding{
+				{
+					Probe:   "pinsDependencies",
+					Outcome: finding.OutcomeNegative,
+					Location: &finding.Location{
+						Type:      finding.FileTypeText,
+						Path:      "test-file",
+						LineStart: &testLineStart,
+						LineEnd:   &testLineEnd,
+						Snippet:   &testSnippet,
 					},
-					Type:   checker.DependencyUseTypeGHAction,
-					Pinned: asBoolPointer(true),
-				},
-			},
-			expected: scut.TestReturn{
-				Error:         nil,
-				Score:         10,
-				NumberOfWarn:  0,
-				NumberOfInfo:  1,
-				NumberOfDebug: 0,
-			},
-		},
-		{
-			name: "GitHub Actions ecosystem with third-party pinned",
-			dependencies: []checker.Dependency{
-				{
-					Location: &checker.File{
-						Snippet: "other/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675",
+					Values: map[string]int{
+						"dependencyType": 6, // pip type
 					},
-					Type:   checker.DependencyUseTypeGHAction,
-					Pinned: asBoolPointer(true),
 				},
-			},
-			expected: scut.TestReturn{
-				Error:         nil,
-				Score:         10,
-				NumberOfWarn:  0,
-				NumberOfInfo:  1,
-				NumberOfDebug: 0,
-			},
-		},
-		{
-			name: "GitHub Actions ecosystem with GitHub-owned and third-party pinned",
-			dependencies: []checker.Dependency{
 				{
-					Location: &checker.File{
-						Snippet: "actions/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675",
+					Probe:   "pinsDependencies",
+					Outcome: finding.OutcomeNegative,
+					Location: &finding.Location{
+						Type:      finding.FileTypeText,
+						Path:      "test-file",
+						LineStart: &testLineStart,
+						LineEnd:   &testLineEnd,
+						Snippet:   &testSnippet,
 					},
-					Type:   checker.DependencyUseTypeGHAction,
-					Pinned: asBoolPointer(true),
-				},
-				{
-					Location: &checker.File{
-						Snippet: "other/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675",
+					Values: map[string]int{
+						"dependencyType": 6, // pip type
 					},
-					Type:   checker.DependencyUseTypeGHAction,
-					Pinned: asBoolPointer(true),
 				},
 			},
-			expected: scut.TestReturn{
-				Error:         nil,
-				Score:         10,
-				NumberOfWarn:  0,
-				NumberOfInfo:  2,
-				NumberOfDebug: 0,
+			result: scut.TestReturn{
+				Score:        0,
+				NumberOfWarn: 2,
+				NumberOfInfo: 1,
 			},
 		},
 		{
-			name: "GitHub Actions ecosystem with GitHub-owned and third-party unpinned",
-			dependencies: []checker.Dependency{
-				{
-					Location: &checker.File{
-						Snippet: "actions/checkout@v2",
+			name: "2 unpinned dependencies for 1 ecosystem shows 2 warn messages",
```

Comment:
some sort of copy/paste error here.

1 ecosystem -> 2 ecosystem maybe?

Reply:
Thanks! https://github.com/ossf/scorecard/pull/3829/commits/dd07fede89c1a68fbf377d8a8c0b9db933700247